packages/docs/src/pages/en/styles/display.md

```diff
@@ -73,6 +73,18 @@ The _condition_ applies the class base on:
 
 Additionally, **media types** can be targeted using the `only` condition. Both `hidden-screen-only` and `hidden-print-only` are currently supported.
 
+### Drawbacks
```

Comment:
This can be placed in an alert under a section ##Caveats which can be placed right above Line 20.
See<https://github.com/vuetifyjs/vuetify/blob/4f49f08e5a86cf9f2f3c07a84cdef5aa53b39545/packages/docs/src/pages/en/components/grids.md?plain=1#L55> for example

Reply:
If you take a look at the updated code now, is that what you meant?